Automattic/wp-calypso
# LiB flow: set the site title, tagline and logo url for every patterns

#### Proposed Changes

* Update the content of LiB patterns
* Set the site title, tagline and logo URL to each pattern

https://user-images.githubusercontent.com/13596067/187895167-cdf80758-8fc6-4b5f-8666-45e021c8ec70.mov

#### Testing Instructions



* Apply D87019-code to your sandbox
* Go through the LiB flow via http://calypso.localhost:3000/setup?flow=link-in-bio
* Check the title, tagline, and logo on each pattern are correct

#### Pre-merge Checklist






Related to #
